### ISSUE TITLE
Git To Da Choppa wrong action point drain

### ISSUE THREAD

**Issue by DaWAAAGH** [WarEmu/WarBugs#20133]
Choppa ability "Git To Da Choppa" drain wrong amount of action points when casted. It drain 20 AP / Sec instead of 10 AP / Sec, like mentioned on ability tooltip.
![GTDC](https://user-images.githubusercontent.com/16031857/170363282-e4c15317-9c62-4154-82f1-a86269e75879.png)

**Comment by DaWAAAGH** [comment]
yes without any interruption the ap cost is 20 instead of 10 now

**Comment by Turkeyshooter** [comment]
Channels drain AP when they tick afaik, so it probably ticks for 20 every 2 seconds, as opposed to 10 every second.
